java/ql/src/semmle/code/java/PrintAst.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Print AST
```

Comment:
Is this file needed for anything?

Reply:
This is the file that the qlref of the test cases points to

Reply:
Ah, yes of course. Quite useful indeed.